test/infrastructure/docker/Makefile

```diff
@@ -45,9 +45,13 @@ ifneq ($(abspath $(ROOT_DIR)),$(shell go env GOPATH)/src/sigs.k8s.io/cluster-api
 endif
 
 # Binaries.
-CONTROLLER_GEN := $(TOOLS_BIN_DIR)/controller-gen
-CONVERSION_GEN := $(TOOLS_BIN_DIR)/conversion-gen
-GOLANGCI_LINT := $(TOOLS_BIN_DIR)/golangci-lint
+CONTROLLER_GEN := $(abspath $(TOOLS_BIN_DIR)/controller-gen)
+CONVERSION_GEN := $(abspath $(TOOLS_BIN_DIR)/conversion-gen)
+GOLANGCI_LINT := $(abspath $(TOOLS_BIN_DIR)/golangci-lint)
+KUSTOMIZE := $(abspath $(TOOLS_BIN_DIR)/kustomize)
+
+$(KUSTOMIZE): # Build kustomize from tools folder.
+	cd $(ROOT); make kustomize
```

Comment:
nit: you can use `$(MAKE) -C $(ROOT) kustomize`

Reply:
Changed.

Reply:
Thank you for making the change.